package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@matrix-org/react-sdk-module-api",
-    "version": "2.3.0",
+    "version": "2.4.0",
```

Comment:
Why are you bumping this? Its for the release automation to do.

https://github.com/matrix-org/matrix-react-sdk-module-api/blob/main/.github/workflows/release.yaml#L33-L39

Reply:
Will fix